e2e/testcases/helm_sync_test.go

```diff
@@ -396,15 +396,29 @@ image:
 // are set. Namespace-scoped resources with their namespace field set should be deployed to their set namespace; namespace-scoped
 // resources with their namespace field not set should be deployed to the default namespace.
 // This test will work only with following pre-requisites:
-// Google service account `e2e-test-ar-reader@${GCP_PROJECT}.iam.gserviceaccount.com` is created with `roles/artifactregistry.reader` for accessing images in Artifact Registry.
+// Google service account `e2e-test-ar-reader@${GCP_PROJECT}.iam.gserviceaccount.com` is created with.
```

Comment:
Why's the line about role deleted?

Reply:
Because the permission and IAM policy binding are created during the test.
This is because gcpserviceaccount and k8sserviceaccount tests are going to create and delete those bindings before and after test, so we can't always assume they always exist.

Reply:
I see, I thought this comment was about what the test's going to do.